src/Init/Data/String/Basic.lean

```diff
@@ -59,9 +77,13 @@ def utf8GetAux : List Char → Pos → Pos → Char
   | c::cs, i, p => if i = p then c else utf8GetAux cs (i + c) p
 
 /--
-  Return character at position `p`. If `p` is not a valid position
-  returns `(default : Char)`.
-  See `utf8GetAux` for the reference implementation.
+Returns the character at position `p` of a string.
+
+If `p` is not a valid position, returns `(default : Char)`.
+
+See `utf8GetAux` for the reference implementation.
+* `"abc".get ⟨1⟩ = 'b'`
+* `"abc".get ⟨3⟩ = (default : Char) = 'A'`
```

Comment:
Worth including an example that shows an invalid position within the string, due to multi-byte characters?

Reply:
Good point. I have added another example to `String.get` variants and `String.set` to show this case.